src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @flow
+ * @file Versions Sidebar component
+ * @author Box
+ */
+
+import * as React from 'react';
+
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+
+import { BackButton } from '../../common/nav-button';
+import Button from '../../../components/button';
+import BoxDrive140 from '../../../illustration/BoxDrive140';
+
+import { LoadingIndicatorWrapper } from '../../../components/loading-indicator';
+
+import VersionsMenu from './VersionsMenu';
+import './StaticVersionsSidebar.scss';
+
+type Props = {
+    isLoading: boolean,
+    onUpgradeClick?: () => void,
```

Comment:
Why is this optional? There doesn't seem to be handling for it being null below.

Reply:
great question, a relic from when I was doing this in the original component. It should never be optional or null here. I will update that!